plugins/kubernetes/app/models/concerns/kubernetes/has_status.rb

```diff
@@ -0,0 +1,13 @@
+module Kubernetes
+  module HasStatus
+    STATUSES = %w[created spinning_up live spinning_down dead failed]
+
+    STATUSES.each do |s|
+      define_method("#{s}?") { status == s }
+    end
+  end
+
+  def status=(new_status)
+    super new_status.to_s
+  end
+end
```

Comment:
Good point! I'll move it there. It didn't occur to me, but makes sense, specially because we use the same method for Release and ReleaseDoc, I think.